.github/workflows/releasability-check.yaml

```diff
@@ -0,0 +1,38 @@
+---
+name: Releasability Check
+on:
+  workflow_call: {}
+
+jobs:
+  releasability_check:
+    name: Releasability Check
+    runs-on: ubuntu-20.04
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - name: Vault Secrets
+        id: secrets
+        uses: SonarSource/vault-action-wrapper@d1c1ab4ca5ad07fd9cdfe1eff038a39673dfca64  # tag=2.4.2-1
+        with:
+          secrets: |
+            development/kv/data/burgr github_username | burgrx_username;
+            development/kv/data/burgr github_password | burgrx_password;
+      - uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b # tag=v3.0.2
```

Comment:
Would it be possible to execute the action with standard `uses: SonarSource/gh-action_release/main@v5` and with parameters (without manual checkout)?